pkg/app/src/pages/organisations/index.tsx

```diff
@@ -30,25 +36,25 @@ interface SortOptionsInterface {
 	value: OrganizationOrderByInput
 }
 
-const sortOptions: SortOptionsInterface[] = [
-	{ value: OrganizationOrderByInput.MemberLimitDesc, name: 'Member: High-Low' },
-	{ value: OrganizationOrderByInput.MemberLimitAsc, name: 'Member: Low-High' },
-	{ value: OrganizationOrderByInput.CreatedAtBlockDesc, name: 'Created: Newest first' },
-	{ value: OrganizationOrderByInput.CreatedAtBlockAsc, name: 'Created: Oldest first' },
-]
-
 export function OrganisationPage() {
 	// todo - ahmad - refactor the filters to an object for the query and other filters
 	const [filters, setFilters] = useState('')
-	const [bodyCount, setBodyCount] = useState<number>(15)
-	const [sortOption, setSortOption] = useState<OrganizationOrderByInput>(sortOptions[0].value)
-	const [fetchOrganizations, { loading, error, data }] = useOrganizationsLazyQuery()
-	useEffect(() => {
-		if (error) {
-			console.error('There was an error querying the display values')
-		}
-	}, [error])
+	const { data: featuresData } = useFeaturesQuery({
```

Comment:
You don't have to load the data yourself it is already done by the app provider.
https://github.com/gamedaoco/gamedao-haiku/blob/main/pkg/app/src/provider/app/appProvider.tsx

Reply:
@DarkNebula0  Can you give me an example for using the app provider with the features